test/e2e/applications_under_test.go

```diff
@@ -146,41 +146,12 @@ var (
 )
 
 func rebuildAppUnderTestContainerImages() {
-	rebuildNodeJsApplicationContainerImage()
-	rebuildJvmApplicationContainerImage()
-	rebuildDotnetApplicationContainerImage()
-}
-
-func rebuildNodeJsApplicationContainerImage() {
-	rebuildApplicationContainerImage(workloadNameNodeJs+"-app", applicationPathNodeJs)
-}
-
-func rebuildJvmApplicationContainerImage() {
-	rebuildApplicationContainerImage(workloadNameJvm+"-app", applicationPathJvm)
-}
-
-func rebuildDotnetApplicationContainerImage() {
-	rebuildApplicationContainerImage(workloadNameDotnet+"-app", applicationPathDotnet)
-}
-
-func rebuildApplicationContainerImage(imageName string, applicationPath string) {
-	By(fmt.Sprintf("building the %s image", imageName))
 	Expect(
 		runAndIgnoreOutput(
 			exec.Command(
-				"docker",
-				"build",
-				applicationPath,
-				"-t",
-				imageName,
+				"make",
+				"test-app-images",
```

Comment:
Image builds for the actual operator images (operator-controller, instrumentation, collector, ...) are no longer triggered by the e2e tests. Instead, they need to have been built before and need to be present or pullable from the target cluster.

However, building auxiliary images like test applications and the Dash0 API mock still are triggered from within the e2e tests --  for now. We can discuss whether we want to remove this make call in a follow up PR, or whether we want to make it optional via an environment variable switch. It will come up again once we move towards running the e2e tests in a remote cluster, where the test app images also need to be present in some registry that the cluster can pull from.